zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -966,7 +966,7 @@ angular.module('zeppelinWebApp')
       }
       return '';
     }
-    var desc = 'Took ' + (timeMs/1000) + ' seconds';
+    var desc = 'Took ' + (timeMs/1000) + ' seconds. Last executed by ' + pdata.authenticationInfo.user + ' at ' + pdata.dateUpdated;
```

Comment:
@felixcheung @AhyoungRyu I agree. I updated the PR. Can you please check if it looks ok? Thanks.